fec-template/public/init.js

```diff
@@ -26,3 +27,39 @@ new feedback.Feedback();
 new typeahead.Typeahead('.js-search-input', $('.js-search-type').val());
 
 new typeaheadFilter.TypeaheadFilter('.js-typeahead-filter', typeahead.datasets.committees);
+
+$('.js-accordion').each(function(){
+  var contentPrefix = $(this).data('content-prefix') || 'accordion';
+  var openFirst = $(this).data('open-first') || false;
+  var opts = {
+    contentPrefix: contentPrefix,
+    openFirst: openFirst
+  };
+  new accordion.Accordion({}, opts);
```

Comment:
Yeah I think we should disable it. It's how we've been initializing our components elsewhere.